plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheckTask.kt

```diff
@@ -39,7 +39,9 @@ open class KtlintCheckTask @Inject constructor(
 
     @get:PathSensitive(PathSensitivity.RELATIVE)
     @get:InputFiles
-    internal val editorConfigFiles: FileCollection = getEditorConfigFiles(project)
+    internal val editorConfigFiles: FileCollection by lazy(LazyThreadSafetyMode.NONE) {
+        getEditorConfigFiles(project)
+    }
```

Comment:
Can you leave a comment explaining why this is intentionally `lazy`? Other than that, this PR looks good!

Reply:
Added comment and force pushed.